packages/api/core/src/util/determine-author.ts

```diff
@@ -0,0 +1,26 @@
+import childProcess from 'child_process';
+import debug from 'debug';
+import { PackagePerson } from '@electron-forge/shared-types';
+import { promisify } from 'util';
+import username from 'username';
+
+const d = debug('electron-forge:determine-author');
+const exec = promisify(childProcess.exec);
+
+const execAndTrimResult = async (command: string) => {
+  const { stdout } = await exec(command);
+  return stdout.trim();
+};
+
+const getAuthorFromGitConfig = async (): Promise<PackagePerson> => {
+  try {
+    const name = await execAndTrimResult('git config --get user.name');
```

Comment:
What directory is this running in, it'll default to `cwd` but that isn't the root of the forge application

Reply:
Thanks, fixed in https://github.com/electron-userland/electron-forge/pull/920/commits/04a7366b9b9673f2bb3bca00e02961a4aafbf68e